bonfire/bonfire.py

```diff
@@ -208,6 +208,13 @@ def _validate_reservation_duration(ctx, param, value):
         help="Duration of the reservation",
         callback=_validate_reservation_duration,
     ),
+    click.option(
+        "--pool-type",
+        type=str,
```

Comment:
If there's only a limited set of valid choices ('default' or 'minimal') -- check out the click 'choice' type: https://click.palletsprojects.com/en/8.1.x/options/#choice-options

Reply:
Cool, just added this.